packages/core/src/types.ts

```diff
@@ -24,12 +24,14 @@ export interface FlatbreadConfig {
   source: Source;
   transformer?: Transformer | Transformer[];
   content: Content;
+  fieldTransform?: (field: string) => string;
```

Comment:
perhaps `fieldNameTransform` would be a clearer name for this as (for me) it semantically collides a bit with field overrides